modules/twinklespeedy.js

```diff
@@ -473,12 +414,12 @@ Twinkle.speedy.generateCsdList = function twinklespeedyGenerateCsdList(list, mod
 			return null;
 		}
 
-		if (criterion.subgroup && !hasSubmitButton) {
+		if (criterion.subgroup && mode.isRadioClick) {
```

Comment:
`mode.isRadioClick` does what it says on the tin, but it isn't an accurate replacement for `!hasSubmitButton`.  I believe this should be
```suggestion
		if (criterion.subgroup && !mode.isMultiple && mode.isRadioClick) {
```
As is, selecting multiple options is broken in `radioClick`.

Reply:
nice catch, weirdly I thought radioclick mode === the one without submit button.

Reply:
It is!  But when there are multiple options, it obviously has to get one.